requirements.txt

```diff
@@ -1,7 +1,8 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-git+https://github.com/canonical/operator/#egg=ops
+#git+https://github.com/canonical/operator/#egg=ops
+git+https://github.com/sed-i/operator@bugfix/make_dirs#egg=ops
```

Comment:
Temporary fix until [operator/674](https://github.com/canonical/operator/pull/674) is resolved.

Reply:
Resolved and now part of `ops` 1.3.0